js/components/conceptsetmodal/conceptSetSaveModal.html

```diff
@@ -1,19 +1,24 @@
-<div class="modal fade" data-bind="modal: show" tabindex="-1" role="dialog">
-    <div class="concept-set-modal modal-dialog" role="document">
-        <div class="modal-header">
-            <i class="fa fa-save"></i> Save New Concept Set
-        </div>
-        <div class="concept-set-dialog modal-body">
-            <span class="help-block" data-bind="visible: notUniqueName"><i class="fa fa-warning"></i> Concept Set already exists</span>
-            <div class="concept-set-form">
-                <label>Concept Set Name:</label>
-                <input type="text" placeholder="Concept Set Name" data-bind="textInput: conceptSetName" />
-            </div>
-        </div>
-        <div class="concept-set-dialog modal-footer">
-            <span class="input-group-btn">
-               <button type="button" data-bind="click: onSave, enable: canSave" class="btn btn-success">Save</button>
-            </span>
+<atlas-modal params="{
+        showModal: $component.show,
+        iconClass: 'fa fa-save',
+        title: 'Save New Concept Set',
+        template: 'conceptSetSaveTmpl',
+        fade: $component.fade,
+        data: $component
+    }">
+</atlas-modal>
+
+<script type="text/html" id="conceptSetSaveTmpl">
```

Comment:
I believe we implemeted the modal type to allow you to embed the template between the `<atlas-modal>` tags. Can you try that out?  having a separate script tag that passes in the template vs just in-line the template in my opinion looks better.  Just remove the `template` param from the component and put the HTML directly inside.